Convenus/ExchangeServiceHelper.cs

```diff
@@ -284,5 +284,25 @@ public class CalendarEvent
         public DateTime EndTime { get; set; }
         public string Subject { get; set; }
 
+    }
+
+    public enum RoomStatus

```

Comment:
Thoughts on having it
```
public enum RoomStatus
{
    Default = 0,
    Taken = 1,
    Available = 2,
    EndOfMeeting = 3
}
```
so that the client can then choose what colors represent each of those statuses?